### PR TITLE
feat(card): update breakpoint sizes

### LINKED ISSUE #1826: [Card] breakpoint sizes are not consistent with latest UX designs
### What package is this for?

- [x] React
- [ ] Angular

### Describe the bug

New dashboard breakpoint sizes have been released

https://ibm.ent.box.com/notes/718007286034?s=xcgyzxuwlws14mput7k8lq4ykz7ui1uu

### Expected behavior

Should use new breakpoints

### PR DESCRIPTION
**Pull request by sstone2423**
Closes #1826 

**Summary**

- Breakpoint sizes have been updated with the latest dashboard guild guidance
- CardEditForm now uses the currently selected breakpoint to determine the card size test

See https://ibm.ent.box.com/notes/718007286034?s=xcgyzxuwlws14mput7k8lq4ykz7ui1uu for the new breakpoints

**Change List (commits, features, bugs, etc)**

- Updated constants for breakpoints
- Updated TimeSeriesCard custom stroke and fill functions with jsdoc comments

**Acceptance Test (how to verify the PR)**

- Best place to test is in any of the DashboardEditor stories so thats its a bit more clear as to what the breakpoint is
- Add and select any card type
- Drag the window (or if you're in the breakpointSwitcher story, swap the breakpoint) and see that the card sizes reflect the new breakpoint guidance
- Also see that the card sizes are reflecting the new sizes based on the current breakpoint in the card size dropdown

### CODE REVIEW

File: src/components/Dashboard/DashboardGrid.jsx [hunk 1]
```diff
@@ -184,7 +184,15 @@ export const findLayoutOrGenerate = (layouts, cards, supportedLayouts) => {
 
     const layoutWithResizableItems = layout.map((cardFromLayout) => {
       const matchingCard = find(cards, { id: cardFromLayout.i });
-      return { ...cardFromLayout, isResizable: matchingCard.isResizable };
+      return {
+        ...cardFromLayout,
+        // only add resizable attribute if it exists
+        ...(matchingCard.isResizable
```
Comment (bjornalm) [inline src/components/Dashboard/DashboardGrid.jsx:190]: Why was this needed? If using `undefined` was problematic we could perhaps use `isResizable: !!matchingCard.isResizable` instead of conditionally excluding the property.

**Review by bjornalm — CHANGES_REQUESTED** [review]
The resize cards story is broken

https://deploy-preview-1827--carbon-addons-iot-react.netlify.app/?path=/story/watson-iot-dashboard-grid--dashboard-all-cards-as-resizable

File: src/components/Dashboard/DashboardGrid.jsx (hunk 1 above)
Comment (sstone2423) [reply src/components/Dashboard/DashboardGrid.jsx:190]: Its not a problem. I just saw it getting added in calls. I'll add it back in

**Review by bjornalm — APPROVED** [review]
LGTM

**Review by tay1orjones — APPROVED** [review]
